Assets/Scripts/Combat/Gun.cs

```diff
@@ -26,6 +35,18 @@ void Update()
         {
             if (Input.GetButton("Fire1"))
             {
+                GameObject Temporary_Bullet_Handler;
+                Temporary_Bullet_Handler = Instantiate(Bullet, Bullet_Emitter.transform.position, Bullet_Emitter.transform.rotation) as GameObject;
+
+                Temporary_Bullet_Handler.transform.Rotate(Vector3.left * 90);
+
+                Rigidbody Temporary_RigidBody;
+                Temporary_RigidBody = Temporary_Bullet_Handler.GetComponent<Rigidbody>();
+
+                Temporary_RigidBody.AddForce(transform.forward * Bullet_Forward_Force);
+                
+                Destroy(Temporary_Bullet_Handler, 3.0f);
+
```

Comment:
Could this be put in the FireGun method? And I don't think you need to create a new game object each time this is called. You could just make them a serializedfield and instantiate them when the FireGun method is called.